lib/rufus/lua/state.rb

```diff
@@ -65,7 +62,7 @@ def fetch_library_method(s)
     def loadstring_and_call(s, bndng, filename, lineno)
 
       bottom = stack_top
-      chunk = filename ? "#{filename}:#{lineno}" : 'line'
```

Comment:
This is for the DEBUG_INTERFACE: https://www.lua.org/manual/5.3/manual.html#lua_Debug

> source: the name of the chunk that created the function. If source starts with a '@', it means that the function was defined in a file where the file name follows the '@'. If source starts with a '=', the remainder of its contents describe the source in a user-dependent manner. Otherwise, the function was defined in a string where source is that string.